docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,27 @@ The ACS will be used for V&V testing. It is defined as:
 - **Not** longitudinal (independent samples) 
 - Includes mail/online, telephone, and personal visits 
 
+**Survey - Current Population Survey (CPS)** 
+
+The CPS is a survey run by the Census Bureau and gathers data about the 
+labor force, employment and unemployment, demographics, earnings, and 
+more information. It is an important survey and therefore is being 
+added here. 
+
+To create this survey: 
+
+- Sample rate of (60,000 / 350 million)% 
+- Sample will be stratified by state to ensure an even distribution 
+- Sample at each time step (approximates monthly) for 12 time steps 
```

Comment:
if the simulation is to run longer than 12 timesteps, which 12 timesteps should be sampled from?

Reply:
Good question - I have made this indefinite (through the length of the simulation) rather than for a defined year. I think this could be something we adjust in scenarios if we don't need all of the information though.